scripts/linking.py

```diff
@@ -358,7 +358,8 @@ def eval_candidate_generation_and_linking(examples: List[data_util.MedMentionExa
                                           k_list: List[int],
                                           thresholds: List[float],
                                           use_gold_mentions: bool,
-                                          spacy_model: str):
+                                          spacy_model: str,
+                                          generate_linking_classifier_training_data: bool):
```

Comment:
If you call this function with multiple k or thresholds, it will duplicate the linking classifier data. Please could you prevent that, preferably by pulling out this functionality into a separate function which only accepts a single k/threshold.

Reply:
added a check